official/vision/beta/projects/yolo/modeling/decoders/yolo_decoder.py

```diff
@@ -396,7 +399,7 @@ def get_raw_depths(self, minimum_depth, inputs):
     depths = []
     if len(inputs.keys()) > 3 or self._fpn_filter_scale > 1:
       for i in range(self._min_level, self._max_level + 1):
-        depths.append(inputs[str(i)][-1] * 2)
+        depths.append(inputs[str(i)][-1]) # * 2)
```

Comment:
Is that intended? Is that going to affect previous experiments? Also remove commented part.